### ISSUE TITLE
Add authentication to btrdbcp

### ISSUE THREAD

**Issue by looselycoupled** [immesys/smartgridstore#12]
If specific perms are required to view streams, then `btrdbcp` is unable to see them in order to copy data to another server.  This issue is to add the ability to add authentication to the btrdcp utility (not in the config file) so that restricted streams can be copied.

It is assumed that wherever streams are being copied to, it is the responsibility of the target server administrator to make sure local permissions are setup to maintain secure access.

**Comment by immesys** [comment]
I have added `$FROM_API_KEY` and `$TO_API_KEY` support. Could you test it out for me?

**Comment by looselycoupled** [comment]
Yes, I should be able to get to it today and get back to you here on the issue.

**Comment by looselycoupled** [comment]
@immesys looks like its working just fine.  I got a warning because I forgot to change the port in the config to 4411 but that's my fault.  Feel free to close this one out.
